src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,25 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        Bulldozer bulldozer = new Bulldozer();
+
+        Excavator excavator = new Excavator();
+
+        Truck truck = new Truck();
+
+        bulldozer.doWork();
+
+        excavator.doWork();
+
+        truck.doWork();
+
+        bulldozer.stopWork();
+
+        excavator.stopWork();
+
+        truck.stopWork();
```

Comment:
Давай уберем все ненужные пустые строки. [Чеклист](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop): Use abstract reference instead of specific one where possible. -> давай использовать абстракцию Machine при создании объектов bulldozer, excavator, truck. И давай создадим массив из машин и будем вызывать методы doWork() и stopWork() в цикле